### ISSUE TITLE
Action cooldown is not working

### ISSUE THREAD

**Issue by Clauvin** [Clauvin/GGJ2017-Waves#3]
The actions in game should work with a cooldown system: you issue an order, the order is executed, the effects are immediate, then the order enters in cooldown. When the cooldown ends, the order is ready to be used again.

The orders' cooldown, as the game is now, are infinite: each order can only be used once.

**Comment by Clauvin** [comment]
(Partially fixed in [#3](https://github.com/Clauvin/GGJ2017-Waves/commit/593b6164f4e030237e6e590888cdbed9027e399a))

The good news: now the passage of one week activates a foreach that checkIfCooledDown() every single PlayerAction in the ActionsManager.

The bad news: as it is, the cooldown will end skipping a week in almost all circunstances, since checkIfCooledDown() will in most cases check the time passed in a week, there will be time equal to less than one week to pass yet, and the next check will be made only in the next week.

In other words:

_Week 1: use an Action between Week 1 and Week 2, cooldown time = 2 weeks.
Week 2: cooldown time = 1,5 weeks. Not yet...
Week 3: cooldown time = 0,5 weeks. Not yet...
Week 4: cooldown time = -0,5 weeks._ **Ok, the action can be used again!**

Conclusion: an action that was used in the middle of the Week 1 and cooldown of two weeks can only be used again **in the start of Week 4, not the Week 3.**

I will see how much of a problem this is later.

**Comment by Clauvin** [comment]
Not much of a problem in the end. Actions just have to be made with a cooldown of one less week. Since no action takes planed zero weeks, all is fine.
